rocksdb_admin/admin_handler.cpp

```diff
@@ -736,6 +759,17 @@ bool AdminHandler::restoreDBHelper(const std::string& db_name,
     return false;
   }
 
+  DBMetaData meta;
+  if (!DecodeThriftStruct(backup_infos.back().app_metadata, &meta)) {
```

Comment:
Is this safe if no app_metadata was backed up?

Reply:
It the app_metadata is empty, then, DBMetaData will be default constructed. This is ok, bcz, when we "writeMetaData", we write with local "db_name".